include/oneapi/tbb/detail/_config.h

```diff
@@ -127,6 +127,14 @@
     #define TBB_USE_ASSERT TBB_USE_DEBUG
 #endif // TBB_USE_ASSERT
 
+#ifndef __TBB_STATISTICS
+#if __TBB_STATISTICS_SUM
+    #define __TBB_STATISTICS 1
+#else
+    #define __TBB_STATISTICS 0
+#endif // __TBB_STATISTICS_SUM
+#endif // __TBB_STATISTICS
```

Comment:
Shouldn't it be like this:
```suggestion
#ifndef __TBB_STATISTICS
    #define __TBB_STATISTICS 0
#endif // __TBB_STATISTICS

#if __TBB_STATISTICS
    #ifndef __TBB_STATISTICS_SUM
        #define __TBB_STATISTICS_SUM 0
     #endif // __TBB_STATISTICS_SUM
#endif
```

Reply:
No, because if you set flag `__TBB_STATISTICS_SUM` we must set also flag `__TBB_STATISTICS`